test.py

```diff
@@ -395,10 +395,7 @@ def __init__(self, path, cfg, options: argparse.Namespace, mode) -> None:
         super().__init__(path, cfg, options, mode)
 
     def build_test_list(self) -> List[str]:
-        """For CQL tests, search for directories recursively"""
-        path = self.suite_path
-        cqltests = itertools.chain(path.rglob("*_test.cql"), path.rglob("test_*.cql"))
-        return [os.path.splitext(t.relative_to(self.suite_path))[0] for t in cqltests]
+        return TestSuite.build_test_list(self)
```

Comment:
If I understand correctly, this changes the globbing to `*test.cql`. I guess it's fine because this is what the cql tests look like anyway. But the commit message is misleading - it's not just repetition, it's actually a different pattern.